### PR TITLE
Add test for making a duplicate submission

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: econplayground/api/tests/test_views.py [hunk 1]
```diff
@@ -161,14 +161,35 @@ def test_create(self):
...
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(Submission.objects.count(), 1)
+
+        self.client.post('/api/submissions/', {
+            'graph': self.g.pk,
+            'choice': 4,
+        })
+
+        # Somehow, the ValidationError is getting swallowed and not
+        # setting the status_code to 400.
```
Comment (nikolas) [inline econplayground/api/tests/test_views.py]: Note to self: the ValidationError is properly displayed in the djangorestframework admin UI, but not in the tests, or when actually tested in javascript with XHR. In practice, this isn't a big deal - the database will throw an IntegrityError when it detects a duplicate, and all this changes is where and how we handle which error.
